Makefile

```diff
@@ -5,17 +5,15 @@
 
 # Configuration of make process in ecos.mk
 include ecos.mk
-CFLAGS += -Iinclude -Iexternal/ldl/include -Iexternal/amd/include -Iexternal/SuiteSparse_config 
+CFLAGS += -Iinclude -Iexternal/qdldl/include -Iexternal/amd/include -Iexternal/SuiteSparse_config
 TEST_INCLUDES = -Itest -Itest/generated
 
 # Compile all C code, including the C-callable routine
 .PHONY: all
 all: libecos.a libecos_bb.a runecos runecosexp
 
-# build Tim Davis' sparse LDL package
-$(LDL):
-	( cd external/ldl    ; $(MAKE) )
-	$(AR) -x external/ldl/libldl.a
+external/qdldl/build/out/libqdldl.a:
```

Comment:
We could also figure out how to build qdldl directly into the ecos library .so and .a files. This way we won't have to change the build and installation process for ecos by having users install libqdldl.so